scripts/openroad/common/io.tcl

```diff
@@ -86,7 +86,17 @@ proc read {args} {
         keys {-override_libs}\
         flags {-multi_corner_libs}
 
-    if {[catch {read_db $::env(CURRENT_ODB)} errmsg]} {
+    if {
+        [catch {
+            if { [info exists ::env(DEF_IN)] } {
+                puts "Reading def $::env(DEF_IN)"
```

Comment:
Remove debug vestiges?

Reply:
I left "Reading odb" because openroad doesn't print the input odb file while it does print the def and lef